src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random random = new Random();
+    private static ColorSupplier colorSupplier;
+
+    private ColorSupplier getColorSupplier() {
+        if (colorSupplier == null) {
+            colorSupplier = new ColorSupplier();
+        }
+        return colorSupplier;
+    }
+
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String randomColor = String.valueOf(colorSupplier.getRandomColor());
+        int randomNumber = random.nextInt(100);

```

Comment:
1) magic number, learn what magic number is
2) random.nextInt() returns 0 inclusively and 100 exclusively